recipes/archicad-apidevkit/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conan import ConanFile
+from conan.tools.files import get
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.00"
+
+
+class ArchicadApidevkitConan (ConanFile):
+    name = "archicad-apidevkit"
+    description = "The General API Development Kit enables software developers to extend the functionality of Archicad"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://archicadapi.graphisoft.com/"
+    license = "https://archicadapi.graphisoft.com/documentation/license-agreement"
+    settings = "os", "arch"
+    no_copy_source = True
+    topics = "API, Archicad, development"
+
+    @property
+    def _source_subfolder(self):
+        return self.source_folder
+
```

Comment:
```suggestion
```

Deprecated, we are moving to Conan 2.0 and `layout()` fits much better. Read more here: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-layout-method